lib/app.js

```diff
@@ -43,169 +42,113 @@ function app (opts) {
     fun: Boolean
   },
-  IAPRange: ['ds:5', 0, 12, 12, 0],
-  size: ['ds:8', 0],
+  IAPRange: ['ds:4', 1, 2, 19, 0],
+  /* size: ['ds:8', 0], */
   androidVersion: {
-    path: ['ds:8', 2],
-    fun: normalizeAndroidVersion
+    path: ['ds:4', 1, 2, 140, 1, 1, 0, 0, 1],
+    fun: helper.normalizeAndroidVersion
   },
-  androidVersionText: ['ds:8', 2],
-  developer: ['ds:5', 0, 12, 5, 1],
+  androidVersionText: {
+    path: ['ds:4', 1, 2, 140, 1, 1, 0, 0, 1],
+    fun: (version) => version || 'Varies with device'
+  },
+  developer: ['ds:4', 1, 2, 68, 0],
   developerId: {
-    path: ['ds:5', 0, 12, 5, 5, 4, 2],
+    path: ['ds:4', 1, 2, 68, 1, 4, 2],
+    fun: (devUrl) => devUrl.split('id=')[1]
+  },
+  developerEmail: ['ds:4', 1, 2, 69, 1, 0],
+  developerWebsite: ['ds:4', 1, 2, 69, 0, 5, 2],
+  developerAddress: ['ds:4', 1, 2, 69, 2, 0],
+  privacyPolicy: ['ds:4', 1, 2, 99, 0, 5, 2],
+  developerInternalID: {
+    path: ['ds:4', 1, 2, 68, 1, 4, 2],
     fun: (devUrl) => devUrl.split('id=')[1]
   },
-  developerEmail: ['ds:5', 0, 12, 5, 2, 0],
-  developerWebsite: ['ds:5', 0, 12, 5, 3, 5, 2],
-  developerAddress: ['ds:5', 0, 12, 5, 4, 0],
-  privacyPolicy: ['ds:5', 0, 12, 7, 2],
-  developerInternalID: ['ds:5', 0, 12, 5, 0, 0],
-  genre: ['ds:5', 0, 12, 13, 0, 0],
-  genreId: ['ds:5', 0, 12, 13, 0, 2],
+  genre: ['ds:4', 1, 2, 79, 0, 0, 0],
+  genreId: ['ds:4', 1, 2, 79, 0, 0, 2],
   familyGenre: ['ds:5', 0, 12, 13, 1, 0],
   familyGenreId: ['ds:5', 0, 12, 13, 1, 2],
-  icon: ['ds:5', 0, 12, 1, 3, 2],
-  headerImage: ['ds:5', 0, 12, 2, 3, 2],
+  icon: ['ds:4', 1, 2, 95, 0, 3, 2],
+  headerImage: ['ds:4', 1, 2, 96, 0, 3, 2],
   screenshots: {
-    path: ['ds:5', 0, 12, 0],
+    path: ['ds:4', 1, 2, 78, 0],
     fun: (screenshots) => {
       if (screenshots === null) return [];
       return screenshots.map(R.path([3, 2]));
     }
   },
-  video: ['ds:5', 0, 12, 3, 0, 3, 2],
-  videoImage: ['ds:5', 0, 12, 3, 1, 3, 2],
-  contentRating: ['ds:5', 0, 12, 4, 0],
-  contentRatingDescription: ['ds:5', 0, 12, 4, 2, 1],
+  video: ['ds:4', 1, 2, 100, 0, 0, 3, 2],
+  videoImage: ['ds:4', 1, 2, 100, 1, 0, 3, 2],
+  contentRating: ['ds:4', 1, 2, 9, 0],
+  contentRatingDescription: ['ds:4', 1, 2, 9, 2, 1],
   adSupported: {
-    path: ['ds:5', 0, 12, 14, 0],
+    path: ['ds:4', 1, 2, 48],
     fun: Boolean
   },
-  released: ['ds:5', 0, 12, 36],
+  released: ['ds:4', 1, 2, 10, 0],
   updated: {
-    path: ['ds:5', 0, 12, 8, 0],
+    path: ['ds:4', 1, 2, 145, 0, 1, 0],
     fun: (ts) => ts * 1000
   },
-  version: ['ds:8', 1],
-  recentChanges: ['ds:5', 0, 12, 6, 1],
-  comments: {
-    useServiceRequestId: 'UsvDTd',
-    path: [0],
-    fun: extractComments
+  version: {
+    path: ['ds:4', 1, 2, 140, 0, 0, 0],
+    fun: (val) => val || 'VARY'
   },
-  editorsChoice: {
+  recentChanges: ['ds:4', 1, 2, 144, 1, 1],
+  comments: {
+    path: ['ds:8', 0],
+    isArray: true,
+    fun: helper.extractComments
+  }
+/*   editorsChoice: {
```

Comment:
is this one missing in the new page? if so we should just remove it

Reply:
Just removed it.